pallets/services-payment/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Moondance Labs Ltd.
```

Comment:
I would suggest to move this in `pallets/services-payment/runtime-api/src/` as this runtime api can also be used directly, without rpc.

Reply:
I agree with you, but I followed how it is defined in other pallets (e.g. `stream-paymet/rpc/runtime-api/src/`). We can create a separate PR changing both paths if we agree on it

Reply:
Okay. No problem. Let's do that.